elastic/synthetics-recorder
# [Release] Bump version to `1.0.0`

## Summary

**EDIT:** We encountered some issues with this release as they relate to importing Playwright utils. Synthetics references `playwright-core` and our custom imports are placed in the `playwright` package, so I've written a [`postinstall` script](https://github.com/elastic/synthetics-recorder/pull/415/files#diff-a777d2721ae7398f81d0a30721af61182979d933265f3ff8d2f4aa2616c3eaaf) that overrides the outputted import in `node_modules` as a short term solution. We will need to revise this for the next release.

Additionally, we had to update the [browser download script](https://github.com/elastic/synthetics-recorder/pull/415/files#diff-36382f2a513237c5de6892ea234931d7e2aab49c1611eaafcdb1de7caee13ef9R61) as its interface has changed and we failed to notice until now because it's a pure JS script.

Resolves https://github.com/elastic/synthetics-dev/issues/183.

Increases the version number to `1.0.0`. After merging this we'll add a tag for v1 and attempt a release.

## Implementation details

N/A

## How to validate this change

We should make sure that we don't need any further changes before merging. We should fix https://github.com/elastic/synthetics-recorder/issues/414 before merging this, but as a workaround we can simply decide not to update the committed NOTICE.txt.
